src/python/pants/backend/javascript/subsystems/nodejs_test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+import logging
+
+import pytest
+
+from pants.backend.javascript.subsystems import nodejs
+from pants.backend.javascript.target_types import JSSourcesGeneratorTarget
+from pants.backend.python import target_types_rules
+from pants.core.util_rules import config_files, source_files
+from pants.engine.process import ProcessResult
+from pants.testutil.rule_runner import QueryRule, RuleRunner
+
+logger = logging.getLogger(__name__)
```

Comment:
Leftover? Generally we don't use logging in tests except as a debugging tool

Reply:
Yep, definitely leftover. weird that it didn't get flagged as unused by the linters.

Reply:
Yeah, I think I also use some super aggressive lint settings in my other projects, to flag module-level stuff - but that's also across the repo.